src/transactions/transaction_db.rs

```diff
@@ -947,6 +965,15 @@ impl TransactionDB<SingleThreaded> {
     pub fn cf_handle(&self, name: &str) -> Option<&ColumnFamily> {
         self.cfs.cfs.get(name)
     }
+
+    /// Drops the column family with the given name
+    pub fn drop_cf(&mut self, name: &str) -> Result<(), Error> {
+        if let Some(cf) = self.cfs.cfs.remove(name) {
+            self.drop_column_family(cf.inner, cf)
+        } else {
+            Err(Error::new(format!("Invalid column family: {}", name)))
```

Comment:
It makes clippy happy
```suggestion
            Err(Error::new(format!("Invalid column family: {name}")))
```